tutorials/cookbooks/running_production.md

```diff
@@ -0,0 +1,308 @@
+
+### 2.2. Enable and start service
+
+The service file we just built is all we need for creating our service. Now we must `enable` the service to tell the operating system that we want it to run MeiliSearch at every boot. We can finally `start` the service to make it run inmediately. We can check everything is working smoothly by checking the service `status`.
+
+```bash
+# Set the service meilisearch
+$ systemctl enable meilisearch
+
+# Start the meilisearch service
+$ systemctl start meilisearch
+
+# Verify that the service is actually running
+$ systemctl status meilisearch
+
+-# --- Expected output ---
+● meilisearch.service - MeiliSearch
+   Loaded: loaded (/etc/systemd/system/meilisearch.service; enabled; vendor preset: enabled)
+   Active: active (running) since Fri 2020-04-10 14:27:49 UTC; 1min 8s ago
+ Main PID: 14960 (meilisearch)
+```
+
+At this point, MeiliSearch is installed and running. It is protected from eventual crashes, system restarts, and most of the problems it could find while running. But it is still hidden and protected inside the walls (or firewalls) of your machine, and unreachable from the outside world. You can stop here if all the requests you do to MeiliSearch are done by another application living in the same machine.
+
+But you probably want to open your MeiliSearch to the outside world, and for now, it is isolated. Let's fix that in a safe way.
+
+## Step 3: Secure and finish your setup. Using a Reverse Proxy, domain name and HTTPS
+
+Now, we want to make our brand new MeiliSearch available to be requested from the outside world. But we want to do it safely. For this purpose we are going to use two of the main technologies available on the web: a Reverse Proxy and SSL
+
+### 3.1. Creating a Reverse Proxy with [Nginx](https://www.nginx.com/)
+
+A reverse proxy is basically an application that will handle every communcation between the outside world and your internal applications. In our case, we want to use Nginx to recieve the HTTP requests coming from the outside world, and redirect them to MeiliSearch itself, but internally. When MeiliSearch has done its amazing job, it will communicate its response to Nginx, which will then transfer this response to the user who sent the request. This is a common way to isolate and protect any application by adding a gate-keeper as robust, secure and fast as Nginx, one of the most safe and efficient tools available online, and of course, Open Source!  
+
+> Reverse proxies are very useful for security, performance, scalability and logging reasons. If you are new to Reverse proxies, you may enjoy this article explaining the why and the how of [reverse proxies](https://www.keycdn.com/support/nginx-reverse-proxy)  
+
+Making Nginx work as a proxy server is really simple. First of all we need to install it in our machine.
+
+```bash
+# Install Nginx on Debian
```

Comment:
Maybe remove this line as Debian was mentionned at the beginning of this tutorial. We assume we will be using a Debian 10 server for this tutorial.
```suggestion
```